src/Extension/Resources/config/task/PbcCodesnifferTask.yaml

```diff
@@ -1,24 +0,0 @@
----
-id: "pbc:validation:php:codestyle-check"
```

Comment:
As far as I remember it was a bugfix. Are you sure we need to remove them?

Reply:
the same